opentimelineio/core/composition.py

```diff
@@ -371,7 +371,7 @@ def trimmed_range_of_child(self, child, reference_space=None):
             result_range.start_time += parent_range.start_time
             current = parent
 
-        if not self.source_range:
+        if not self.source_range or not result_range:
```

Comment:
whoops, that bug fix wasn't meant to go into this one.  I can split it out if you want @jminor

Reply:
Yeah, if you could split it out that would keep this clean. Thanks.